test/reflect.js

```diff
@@ -0,0 +1,483 @@
+/*global describe, it, expect, require, Reflect */
+
+var exported = require('../');
+
+describe('Reflect', function () {
+  var object = {
+    something: 1,
+    _value: 0
+  };
+
+  Object.defineProperties(object, {
```

Comment:
These tests need to run in ES3, where `Object.defineProperties` won't be available. Please ensure that any tests that rely on property descriptors being supported are skipped in browsers that don't support them.